contrib/sdwire/pyproject.toml

```
@@ -0,0 +1,28 @@
+[project]
+name = "jumpstarter-driver-sdwire"
+version = "0.1.0"
+description = ""
+authors = [
+  { name = "Miguel Angel Ajo Pelayo", email = "majopela@redhat.com" },
+  { name = "Nick Cao", email = "ncao@redhat.com" },
+  { name = "Kirk Brauer", email = "kbrauer@hatci.com" },
```

```suggestion
  { name = "Nick Cao", email = "ncao@redhat.com" },
```

Let's be honest!, good work!